commitizen/commands/bump.py

```diff
@@ -230,6 +230,13 @@ def __call__(self):  # noqa: C901
             c = git.commit(message, args=self._get_commit_args())
         if c.return_code != 0:
             raise BumpCommitFailedError(f'2nd git.commit error: "{c.err.strip()}"')
+
+        if c.out:
+            out.write("git commit stdout: " + c.out)
```

Comment:
do we need the string prefix? or doing `out.write(c.out)` would be enough?

Reply:
sounds good.

Reply:
done.